guides/source/upgrading_ruby_on_rails.md

```diff
@@ -76,6 +76,16 @@ Rails 5.2 adds bootsnap gem in the [newly generated app's Gemfile](https://githu
 The `app:update` task sets it up in `boot.rb`. If you want to use it, then add it in the Gemfile,
 otherwise change the `boot.rb` to not use bootsnap.
 
+### Expirity is now embedded in the cookies values
+
+To improve security, Rails now embeds the expirity information also in encrypted or signed cookies value.
+
+This new embded informaiton make those cookies incompatible with versions of Rails older than 5.2.
+
+If you require your cookies to be read by 5.1 and older, set
+`Rails.application.config.action_dispatch.embed_expirity_in_the_cookie_value` to `false`.
```

Comment:
Might want to suggest that it's useful for when you're validating your 5.2 deploy and allows you to rollback.